romana-install/roles/provision/create-aws-hosts/tasks/main.yml

```diff
@@ -0,0 +1,27 @@
+---
+- name: Generate CloudFormation Template
+  template: src="aws_cf_small_cluster.json" dest="{{ stack_data_dir }}/cloudformation.json"
```

Comment:
Changing "stacks" to "_tmp" would be fairly trivial, since everything related to it depends on a stack_data_dir variable.
I didn't know about VAGRANT_VAGRANTFILE setting. Could be useful.